src/runtime/Object.cpp

```diff
@@ -519,6 +519,11 @@ ObjectGetResult Object::getOwnProperty(ExecutionState& state, const ObjectProper
     return ObjectGetResult();
 }
 
+ObjectGetResult Object::getOwnProperty(ExecutionState& state, const ObjectPropertyName& propertyName) ESCARGOT_OBJECT_SUBCLASS_MUST_REDEFINE
+{
+    return this->ordinaryGetOwnProperty(state, propertyName);
```

Comment:
Changing `Object::getOwnProperty' to invoke `Object::ordinaryGetOwnProperty` in itself may affect the performance slowdown because of additional function call. IMO it's better to merge these two methods. How about your opinion?

Reply:
Sure, thanks for the remind I also wanted to merge these functions.